src/zhinst/qcodes/qcodes_adaptions.py

```diff
@@ -483,6 +483,8 @@ def tk_node_to_qcodes_list(tk_node: Node):
     else:
         parents = tk_node.raw_tree[:-1]
         name = tk_node.raw_tree[-1]
+        #attributes are not allowed to start with a number (#31)
+        name = "_" + name if name[0].isdigit() else name
```

Comment:
It's a bit unfortunate, since the underscore prefix means it's seen as a protected variable. But I don't have a better suggestion. As far as I can tell, these nodes on HF2 are the only ones existing that begin with a digit. Scope and impact is rather limited, as these nodes are only useful for debugging faulty hardware.

```suggestion
        # Attributes are not allowed to start with a number (#31)
        name = "_" + name if name[0].isdigit() else name
```